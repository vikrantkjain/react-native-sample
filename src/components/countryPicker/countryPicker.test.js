import React from 'react';
import renderer from 'react-test-renderer';
import CountryPicker from './index';

jest.mock('react-native-config', () => 'RNConfig');

test('CountryPicker render correctly', () => {
  const tree = renderer
    .create(<CountryPicker placeholder="country picker" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
