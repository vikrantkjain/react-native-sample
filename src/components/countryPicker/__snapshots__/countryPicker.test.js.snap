// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountryPicker render correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "white",
      "borderColor": "#303030",
      "borderWidth": 2,
      "height": 60,
      "justifyContent": "center",
      "marginBottom": 10,
      "padding": 10,
      "width": 250,
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": false,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "flex": 1,
        "paddingLeft": 5,
        "paddingRight": 16,
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <Text
        style={
          {
            "color": "#000",
            "paddingLeft": 5,
            "paddingRight": 5,
            "textAlign": "right",
          }
        }
      >
        +1
      </Text>
      <View
        style={
          [
            {
              "overflow": "hidden",
            },
            {
              "alignItems": "center",
              "height": 15,
              "justifyContent": "center",
              "marginLeft": 5,
              "paddingRight": 5,
              "width": 15,
            },
          ]
        }
      >
        <FastImageView
          defaultSource={null}
          resizeMode="cover"
          source={
            {
              "testUri": "../../../src/assets/images/ic_drop_down.png",
            }
          }
          style={
            {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
    </View>
  </View>
  <Modal
    hardwareAccelerated={false}
    onRequestClose={[Function]}
    visible={false}
  />
</View>
`;
